doc/devices/qpu.rst

```diff
@@ -64,33 +62,18 @@ These operations include:
     pennylane_forest.ops.ISWAP
     pennylane_forest.ops.PSWAP
 
-Device options
-~~~~~~~~~~~~~~
-
-On initialization, the PennyLane-Forest devices accept additional keyword 
-arguments beyond the PennyLane default device arguments.
-
-``forest_url`` (*str*)
-    the Forest URL server. Can also be set by
-    the environment variable ``FOREST_SERVER_URL``, or in the ``~/.qcs_config``
-    configuration file. Default value is ``"https://forest-server.qcs.rigetti.com"``.
-
-``qvm_url`` (*str*)
-    the QVM server URL. Can also be set by the environment
-    variable ``QVM_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:5000"``.
-
-``compiler_url`` (*str*)
-    the compiler server URL. Can also be set by the environment
-    variable ``COMPILER_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:6000"``.
```

Comment:
Why did you delete these keyword arguments? I believe you talk about them in the `quilc server configuration` right below.

Reply:
Good catch. These kwargs were removed but I didn't update the note to properly reflect that. I've corrected the note in both places.